views/partials/head.php

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Site Name</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+</head>
+<body>
+  <div class="container-fluid">
+    <?php include 'nav.php'; ?>
```

Comment:
we maybe need to make a distinction here between the <head/> of the html and the "header" as seen in the UI. For the sake of simplicity I think it is ok for the <head/> and <body/> to live in the index.php, and let that file serve as our application shell. Then we can add and remove components from the <body/> as needed without worrying about breaking our html structure